### DISCUSSION TITLE
Use PHP as server language?

### DISCUSSION THREAD

**Discussion by KoljaL** [pouchdb/pouchdb#8657] [General]
Hello everybody, 

I'm looking for a way to sync data from a PWA to a website. 
My really big problem is, that there is only PHP available on the server.

Is there any way to use PouchDB?  
Or have you some hints for me, for other libs? 

Regard Kolja

**Comment by SourceR85** [comment]
A rewrite of PouchDB in PHP is very unlikely.

Your best option is to install [CouchDB](https://couchdb.apache.org/) (the project, pouch is derived from) on a separate machine or inside your cloud environment via [docker](https://hub.docker.com/_/couchdb).
